src/test/groovy/edu/ie3/datamodel/io/processor/input/GraphicInputProcessorTest.groovy

```diff
@@ -0,0 +1,74 @@
+package edu.ie3.datamodel.io.processor.input
+
+import edu.ie3.datamodel.models.input.graphics.LineGraphicInput
+import edu.ie3.datamodel.models.input.graphics.NodeGraphicInput
+import edu.ie3.test.common.GridTestData
+import edu.ie3.util.TimeTools
+import spock.lang.Specification
+
+import java.time.ZoneId
+
+class GraphicInputProcessorTest extends Specification {
+    static {
+        TimeTools.initialize(ZoneId.of("UTC"), Locale.GERMANY, "yyyy-MM-dd HH:mm:ss")
+    }
+
+    def "The GraphicInputProcessor should de-serialize a provided NodeGraphicInput with point correctly"(){
+        given:
+        GraphicInputProcessor processor = new GraphicInputProcessor(NodeGraphicInput.class)
+        NodeGraphicInput validNode = GridTestData.nodeGraphicC
+        Map expected = [
+                "uuid"          : "09aec636-791b-45aa-b981-b14edf171c4c",
+                "graphicLayer"  : "main",
+                "path"          : "",
+                "point"         : "{\"type\":\"Point\",\"coordinates\":[0.0,10],\"crs\":{\"type\":\"name\",\"properties\":{\"name\":\"EPSG:4326\"}}}",
+                "node"          : "bd837a25-58f3-44ac-aa90-c6b6e3cd91b2"
+        ]
+
+        when:
+        Optional<LinkedHashMap<String, String>> actual = processor.handleEntity(validNode)
+
+        then:
+        actual.isPresent()
+        actual.get() == expected
+    }
+
+    def "The GraphicInputProcessor should de-serialize a provided NodeGraphicInput with path correctly"(){
+        given:
+        GraphicInputProcessor processor = new GraphicInputProcessor(NodeGraphicInput.class)
+        NodeGraphicInput validNode = GridTestData.nodeGraphicD
+        Map expected = [
+                "uuid"          : "9ecad435-bd16-4797-a732-762c09d4af25",
+                "graphicLayer"  : "main",
+                "path"          : "{\"type\":\"LineString\",\"coordinates\":[[-1,0.0],[1,0.0]],\"crs\":{\"type\":\"name\",\"properties\":{\"name\":\"EPSG:4326\"}}}",
```

Comment:
Why do nodes have a path input?!

Reply:
This meant to be used, if you would like to draw the node as a bus bar (a line). The point is used, if the node is drawn as a point - what surprise 😃.